viser/client/src/WebsocketInterface.tsx

```diff
@@ -417,6 +417,28 @@ function useMessageHandler() {
         );
         return;
       }
+      // Add a camera-aligned RGBD image
+      case "PopupImageMessage": {
+        viewer.popupMaterialRef.current!.uniforms.enabled.value = true;
+        new TextureLoader().load(
```

Comment:
have you done any monitoring on memory? I'm worried there's a leak here, since we don't call `dispose()` on loaded textures

not 100% sure what the best solution is, but we could follow the same pattern as `BackgroundImageMessage` or maybe also look into `useLoader`: https://docs.pmnd.rs/react-three-fiber/tutorials/loading-textures

Reply:
@brentyi fixed in recent commit